src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PreviewWindow.xaml

```diff
@@ -31,14 +31,8 @@
         Height="auto" />
     </Grid.RowDefinitions>
     <TextBlock
-      Grid.Row="0"
-      Margin="12,8,12,0"
-      Text="{x:Static nuget:Resources.Text_ReviewChanges}"
-      FontSize="{Binding ElementName=_self,Path=FontSize,Converter={StaticResource Font122PercentSizeConverter}}" />
-
-    <TextBlock
       Grid.Row="1"
```

Comment:
Since, we no longer have the first text block, we should consider moving this text block to Grid.Row = "0" and move any other items below up by one as well.